core/src/main/java/com/linecorp/armeria/client/ClientRequestContext.java

```diff
@@ -507,6 +508,18 @@ default void timeoutNow() {
         cancel(ResponseTimeoutException.get());
     }
 
+    /**
+     * Returns whether this {@link ClientRequestContext} has been timed-out, that is the cancellation cause
+     * is an instance of {@link TimeoutException} or
+     * {@link UnprocessedRequestException} and wrapped cause is {@link TimeoutException}.
+     */
+    @Override
+    default boolean isTimedOut() {
```

Comment:
I suggested overriding it because `UnprocessedRequestException` occurs on the client side. By the way, should we call `RequestContext.isTimedOut()` and chain it with `||` operator to minimize duplication and not miss changes in super?
```java
@Override
default boolean isTimedOut() {
    if (RequestContext.super.isTimedOut()) {
        return true;
    }
    final Throwable cause = cancellationCause();
    return cause instanceof UnprocessedRequestException && cause.getCause() instanceof TimeoutException;
}
```